docs/upgrade.html

```diff
@@ -144,8 +144,10 @@ <h5><a id="upgrade_350_kraft" href="#upgrade_350_kraft">Upgrading KRaft-based cl
                 ./bin/kafka-features.sh upgrade --metadata 3.5
             </code>
         </li>
-        <li>Note that the cluster metadata version cannot be downgraded to a pre-production 3.0.x, 3.1.x, or 3.2.x version once it has been upgraded.
-            However, it is possible to downgrade to production versions such as 3.3-IV0, 3.3-IV1, etc.</li>
+        <li>Note that cluster metadata downgrade is not supported, although it might be possible to do a safe downgrade in some cases.
```

Comment:
I'm thinking this sentence is too strong: `Note that cluster metadata downgrade is not supported`. 

Maybe:
`Note that cluster metadata downgrade might be possible in some cases. Every <a href="https://github.com/apache/kafka/blob/trunk/server-common/src/main/java/org/apache/kafka/server/common/MetadataVersion.java">MetadataVersion</a> after 3.2.x has a boolean parameter...`

Reply:
What are the cases the downgrade is supported after you do the `./bin/kafka-features.sh upgrade --metadata 3.5` step that directly precedes this? I think this should be worded much more strongly.

```suggestion
        <li>Once you upgrade the metadata version to `3.5`, there is no possibility of downgrade.
```

And I think the same applies to 3.4 and likely also for 3.3 (I did not checked that one in detail).

Reply:
I know it's strong, but I think it reflects user experience. The reality is that metadata is still evolving at a fast pace.

Reply:
Changed to "not supported in this version".